app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<header>
+  <%= link_to(image_tag('back.png', alt: 'Back'), groups_path, class: 'back-button') %>
+  <h1>DETAILS</h1>
+</header>
+<main class="g-bg">
+  <section class="w-bg">
+    <% if @purchases.empty? %>
+      <%= button_to "delete", {:controller => :groups, :action => 'destroy', :id => @group.id }, :method => :delete %>
+    <% end %>
+    <%= link_to group_purchases_url(@group), data: { turbo: false }, class: 'item' do %>
+      <div class="category-card">
+        <div class="category-image-info">
+          <%= image_tag(check_url(@group.icon), class: 'icon', alt: 'group icon') %>
+          <div class="category_info">
+            <span class="name"><%= @group.name %></span>
+          </div>
+        </div>
+      </div>
+    <% end %>    
+    <ul class="purchases-list">
+      <% @purchases.each do |purchase| %>
+        <li>
+          <%= link_to group_purchase_url(@group, purchase) do %>
+            <div class="purchase-details">
+              <span><%= purchase.name %></span>
+              <span>$<%= purchase.amount %></span>
+            </div>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+    <div class="cagetory-total">
+      <span>TOTAL PAYMENT</span>
+      <span>$<%= total_amount(@group) %></span>
+    </div>
+  </section>
+  <section class="below-form">
+    <%= link_to "Add a Transaction", new_group_purchase_path %>
+  </section>
+</main>
```

Comment:
- [ ] Please apply the style changes in the transactions index page as well. For the mobile view. 👍 
![RorBudgetApp - Google Chrome 12_9_2023 1_26_01 PM](https://github.com/firepicaso/ror-budget-app/assets/47625407/d5771c41-8b50-4aec-847a-d87a49067309)